packages/contracts/src/state-dump/make-dump.ts

```diff
@@ -159,6 +159,7 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
     'OVM_ProxyEOA',
     'OVM_ExecutionManagerWrapper',
     'OVM_GasPriceOracle',
+    'OVM_SafetyChecker',
```

Comment:
This is necessary in order to move the safety cache's functionality into the safety checker. Presumably it wouldn't affect the contents of the `isByteCodeSafe` function, but it's something to consider.